apps/passport-server/.env.local.example

```diff
@@ -121,3 +121,6 @@ FROGCRYPTO_ADMIN_USER_EMAILS=
 
 # The base URL for frogcrypto assets
 FROGCRYPTO_ASSETS_URL=
+
+# Disable rate-limiting locally by setting this to "true"
+RATE_LIMIT_DISABLED=
```

Comment:
There are now multiple rate limit environment variable features - the one you've implemented, and the one I implemented for account resets. Longer term, it may be nice to consolidate somehow. I don't think that's possible right now because `RateLimitService`'s implementation lets its users specify rate limits in terms of operations per hour, whereas the account reset rate limit is specified in terms of resets/day.

Anyways, disambiguating the env variables a bit may be nice to prevent confusion for now.

Reply:
I've created an issue for consolidating the rate-limiting https://github.com/proofcarryingdata/zupass/issues/1379

I'll make the environment variables a bit more specific, so as to avoid ambiguity.